apps/meteor/server/models/raw/LivechatDepartmentAgents.ts

```diff
@@ -139,8 +139,8 @@ export class LivechatDepartmentAgentsRaw extends BaseRaw<ILivechatDepartmentAgen
 		return this.find({ departmentId: { $in: departmentIds } }, options);
 	}
 
-	findAgentsByAgentIdAndBusinessHourId(_agentId: string, _businessHourId: string): [] {
-		return [];
+	findAgentsByAgentIdAndBusinessHourId(_agentId: string, _businessHourId: string): Promise<ILivechatDepartmentAgents[]> {
+		return [] as unknown as Promise<ILivechatDepartmentAgents[]>;
```

Comment:
an empty array is not enough?
```suggestion
	async findAgentsByAgentIdAndBusinessHourId(_agentId: string, _businessHourId: string): Promise<ILivechatDepartmentAgents[]> {
		return [];
```